Yoast/wordpress-seo
# P2-342 Feature/schema blocks

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Merges the schema feature branch into trunk. The feature is behind a feature-flag (`YOAST_SEO_SCHEMA_BLOCKS`).

## Relevant technical choices:

* Also had to do 2 small commits on the monorepo to get the travis build to succeed:
  - https://github.com/Yoast/javascript/commit/40f08d843fd3d7280646db04bc531ba05ef5a329
  - https://github.com/Yoast/javascript/commit/aa0fa531a3f0238fbae60bf106c5dcbb92923a1e
* Released the `@yoast/schema-blocks` package, so it won't fail on `trunk`.

## Test instructions

This PR can be tested by following these steps:

* Link the monorepo and make sure you check out the latest `develop` branch on the monorepo.
* add the `YOAST_SEO_SCHEMA_BLOCKS` feature flag to your Wordpress config file. `plugin-development-docker/config`.
* Check that the `Recipe` block is available in the editor.
* Comment out the `YOAST_SEO_SCHEMA_BLOCKS` feature flag to your Wordpress config file.
* Check that the block is no longer available.

## UI changes

## Documentation

## Quality assurance


Fixes #
